src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig

```diff
@@ -1,13 +1,11 @@
-{% extends '@SyliusShop/account/address_book/layout.html.twig' %}
+{% extends '@SyliusShop/shared/layout/base.html.twig' %}
 
 {% block title %}{{ 'sylius.ui.add_address'|trans }} | {{ parent() }}{% endblock %}
 
-{% block account_header %}
-    {% set title = 'sylius.ui.address_book'|trans %}
-    {% set description = 'sylius.ui.add_address'|trans %}
-    {{ parent() }}
-{% endblock %}
-
-{% block account_content %}
-    {% hook 'sylius_shop.account.address_book.create' with { form } %}
+{% block content %}
+    <div class="container mb-auto">
+        <div class="row my-4">
+            {% hook 'layout' with { _prefixes: ['sylius_shop.account.address_book.create', 'sylius_shop.shared.account'], form } %}
```

Comment:
`['sylius_shop.account.address_book.index', 'sylius_shop.shared.account']`
This should be done by setting the `prefixes` variable, similar to how it's done in `src/Sylius/Bundle/ShopBundle/templates/account/order/index.html.twig`. This way manage the 3 hooks (with  `sylius_shop.base.layout`).